src/Hyperlink/index.jsx

```diff
@@ -51,7 +51,7 @@ Hyperlink.defaultProps = {
 
 Hyperlink.propTypes = {
   destination: PropTypes.string.isRequired,
-  content: PropTypes.string.isRequired,
+  content: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
```

Comment:
I want to explicitly call out the addition of the `element` prop type in `Hyperlink`.